src/_pytest/deprecated.py

```diff
@@ -110,3 +113,7 @@
     "pytest/tmpdir_factory.ensuretemp is deprecated, \n"
     "please use the tmp_path fixture or tmp_path_factory.mktemp"
 )
+
+PYTEST_LOGWARNING = PytestDeprecationWarning(
+    "pytest_logwarning is deprecated and will be removed soon, use pytest_warning_captured instead"
```

Comment:
this message should include the critical information that the hook is no longer being called

Reply:
True

Reply:
Fixed! 👍